Tests/UITests/TestApp/ExampleStandard.swift

```diff
@@ -14,19 +14,33 @@ import SwiftUI
 
 /// An example Standard used for the configuration.
 actor ExampleStandard: Standard, EnvironmentAccessible {
-    @Published @MainActor var consentData: PDFDocument = .init()
+    @Published @MainActor var firstConsentData: PDFDocument = .init()
+    @Published @MainActor var secondConsentData: PDFDocument = .init()
 }
 
 
 extension ExampleStandard: OnboardingConstraint {
-    func store(consent: PDFDocument) async {
+    func store(consent: PDFDocument, identifier: String) async {
         await MainActor.run {
-            self.consentData = consent
+            if identifier == "FirstConsentDocument" {
```

Comment:
I have thought a bit about the usage of "magic strings" as identifiers for the project.

I think it still makes sense to use strings as an extensible way to define identifiers and make it possible to load identifiers from a remote system or make them dynamic compared to having a typed solution with, e.g., enum cases.

I would nevertheless suggest that we demonstrate some best practices in the documentation and testing application. One would be to avoid re-declaring the string across the application and rather use an `enum` with no cases to represent the choices and use that across the application where we reference the identifier.

```swift
enum ConsentDocumentIdentifier {
    static let first = "consentDocumentIdentifier1"
    static let second = "consentDocumentIdentifier2"
}
```

This would then turn this statement to:
```suggestion
            if identifier == ConsentDocumentIdentifier.first {
```

Reply:
Done! I added an enum ConsentDocumentIdentifier with identifier strings, instead of using the actual "magic strings" throughout the code.